### PR TITLE
[CN-284] Add section for the HotBackup status

### PR DESCRIPTION
**Pull request by SeriyBg**

### CODE REVIEW

File: docs/modules/ROOT/pages/backup-restore.adoc [hunk 1]
```diff
@@ -116,6 +116,21 @@ kubectl apply -f ./hot-backup-scheduled.yaml
 ----
 --
 
+== Hot Backups Status
```
Comment (JakeSCahill) [inline docs/modules/ROOT/pages/backup-restore.adoc:119]:
```suggestion
== Checking the Status of a Hot Backup
```

File: docs/modules/ROOT/pages/backup-restore.adoc [hunk 2]
```diff
@@ -116,6 +116,21 @@ kubectl apply -f ./hot-backup-scheduled.yaml
...
+
+You can check status of the Hot Backup task by running the folowing command.
```
Comment (JakeSCahill) [inline docs/modules/ROOT/pages/backup-restore.adoc:121]:
```suggestion
To check the status of a hot backup, run the following command:
```

File: docs/modules/ROOT/pages/backup-restore.adoc [hunk 3]
```diff
@@ -116,6 +116,21 @@ kubectl apply -f ./hot-backup-scheduled.yaml
...
+[source,bash]
+----
+kubectl get hotbackup hot-backup
+----
+
+You should see the similar output that will indicate that the Hot Backup process finished successfully.
```
Comment (JakeSCahill) [inline docs/modules/ROOT/pages/backup-restore.adoc:127]:
```suggestion
The status of the hot backup is displayed in the output.
```

What other statuses are there? Can we provide some direction for users if a hot backup fails?

Comment (SeriyBg) [reply docs/modules/ROOT/pages/backup-restore.adoc:127]: The statuses are self descriptive, and are `"Unknown", "NotStarted", "InProgress", "Failure", "Success"`. Should we provide an additional description for them? For me it seems redundant and will like: `Failure - indicates that the Hot Backup task failed` 😃

**Review by hakanmemisoglu — APPROVED** [review]

**Pull request merged**